src/Components/Ignitor/src/BlazorClient.cs

```diff
@@ -544,14 +555,23 @@ private string[] ReadMarkers(string content)
             var markers = matches.Select(s => (value: s.Groups[1].Value, parsed: JsonDocument.Parse(s.Groups[1].Value)))
                 .Where(s =>
                 {
-                    var markerType = s.parsed.RootElement.GetProperty("type");
-                    return markerType.ValueKind != JsonValueKind.Undefined && markerType.GetString() == "server";
+                    return s.parsed.RootElement.TryGetProperty("type", out var markerType) &&
```

Comment:
When using `ServerPrerendered`, we get an instance of the marker at the end which does not have these properties. The previous code throws when it encounters this marker comment.